_posts/2021-05-25-hhvm-4.111.markdown

```diff
@@ -0,0 +1,30 @@
+---
+title: "HHVM 4.111"
+layout: post
+author: jjergus
+category: blog
+---
+
+HHVM 4.111 is released! This release marks the end of support for 4.105;
+HHVM 4.106&ndash;4.110 remain supported, as do the 4.80 and 4.102 LTS releases.
+
+# Highlights
+
+- *Abstract*
+  [context constants](https://docs.hhvm.com/hack/contexts-and-capabilities/context-constants)
+  are now allowed by default (in previous HHVM versions, they can be enabled
+  using the INI option `enable_abstract_context_constants = true`).
+- The typechecker no longer raises an error if the `?->` operator is used on an
+  object of a non-nullable type (these calls are safe, even if unnecessary).
+
+# Breaking Changes
+
+- The typechecker now raises an error when the same non-abstract method is
```

Comment:
would 'multiple traits define non-abstract methods with the same name' be clearer + still accurate?

Reply:
I'm not completely sure but I think the common cases that would best fit that description were already covered. This is based on https://github.com/facebook/hhvm/commit/cc643c432a0ddfcb749da29753c982d8768c1eab where the example is literally the same method.